runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -573,12 +574,331 @@ static TR::Instruction *generatePrefetchAfterHeaderAccess(TR::Node
    return instr;
    }
 
+// General float/double convert to long
+//
+TR::Register *J9::X86::TreeEvaluator::fpConvertToLong(TR::Node *node, TR::SymbolReference *helperSymRef, TR::CodeGenerator *cg)
+   {
+   TR::Compilation *comp = cg->comp();
+   TR_ASSERT(cg->comp()->target().is32Bit(), "AMD64 doesn't use this logic");
+
+   TR::Node *child = node->getFirstChild();
+
+   // TODO: Perform f2l using SSE.
```

Comment:
Remove this TODO.  This is not something that can be done cheaply on 32-bit, and we won't consider it.